<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0" rowspan="2" colspan="3">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Do you want to include the handouts in the printing?</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="btnCancel">
            <property name="text">
             <string>No (Next)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btnAccept">
            <property name="text">
             <string>Yes</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>391</width>
            <height>131</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>LOGIN</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <widget class="QLineEdit" name="lineEditPassword">
           <property name="geometry">
            <rect>
             <x>114</x>
             <y>60</y>
             <width>251</width>
             <height>25</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEditUsername">
           <property name="geometry">
            <rect>
             <x>114</x>
             <y>30</y>
             <width>251</width>
             <height>25</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
          </widget>
          <widget class="QLabel" name="lblUsername">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>35</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblPassword">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>65</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblForgotenPassword">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>106</y>
             <width>261</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="btnCancel_2">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>280</y>
            <width>78</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>&amp;Cancel</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnNext">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>280</y>
            <width>78</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>&amp;Next</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_3">
         <widget class="QTableWidget" name="tableWidgetAssignments">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>500</width>
            <height>261</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="btnCancel_3">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>290</y>
            <width>78</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>&amp;Cancel</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnNext_2">
          <property name="geometry">
           <rect>
            <x>420</x>
            <y>290</y>
            <width>78</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>&amp;Next</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_4">
         <widget class="QPushButton" name="btnPrint">
          <property name="geometry">
           <rect>
            <x>390</x>
            <y>230</y>
            <width>70</width>
            <height>70</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
         </widget>
         <widget class="QProgressBar" name="progressBar">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>70</y>
            <width>341</width>
            <height>41</height>
           </rect>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditUsername</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>btnNext</tabstop>
  <tabstop>btnCancel_2</tabstop>
  <tabstop>tableWidgetAssignments</tabstop>
  <tabstop>btnNext_2</tabstop>
  <tabstop>btnCancel_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
